geode-management/src/test/java/org/apache/geode/management/configuration/GatewayReceiverTest.java

```diff
@@ -23,24 +23,24 @@
 import org.apache.geode.management.api.RestfulEndpoint;
 
 
-public class GatewayReceiverConfigTest {
-  private GatewayReceiverConfig receiver;
+public class GatewayReceiverTest {
```

Comment:
I think this unit test should have some coverage of each of the public methods on the class it is testing. It could be as simple as a call of the settor and then validating that the gettor returns what is expected. It currently looks like it is only testing getId and getUri

Reply:
I thought we agreed that we don't test common getter/setters that contains no logic. I have tests for id and uri to make sure group names are used as ids.

Reply:
I think it is worth testing the logic we do have, which is a little bit. For example if you just had one test method that verified that if you call a settor then the corresponding gettor will return what was set then at least we make sure we didn't copy & paste the wrong field names in the methods.

Reply:
these are IDE generated code, it would be a bit mundane to test them.

Reply:
https://stackoverflow.com/questions/6197370/should-unit-tests-be-written-for-getter-and-setters 
looks like the industry is also having debate on this. I would go with the majority :-).